ui/app/utils/default_jobs/actions.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export default `job "job-with-actions" {
+  # Specifies the datacenter where this job should be run
+  # This can be omitted and it will default to ["*"]
+  datacenters = ["*"]
+
+  # Run the job only on Linux or MacOS.
+  constraint {
+    attribute = "\${attr.kernel.name}"
+    operator  = "set_contains_any"
+    value     = "darwin,linux"
+  }
+
+  group "my_group" {
+    # Specifies the number of instances of this group that should be running.
+    # Use this to scale or parallelize your job.
+    count = 3
+
+    task "sleepy" {
+      driver = "raw_exec"
```

Comment:
The `raw_exec` driver is disabled by default on non-dev clusters. Should we use `docker` instead?

The actions also assume things about the node, like having bash, awk, and curl installed, which may not be true.

Reply:
Hm — I was secretly hoping that this could be our first "If you don't have docker installed you can still play around with Nomad" example job, but you're right. Actions with sh/curl/awk is surely not the one to try that out on.

Reply:
Unfortunately that would be very hard to achieve and it's 95% of the motivation behind Docker being created in the first place 😅 

If you want a non-Docker job example I think it should be marked as such with all the important caveats ("you must have X, Y, and Z installed") listed before users run it.